src/server.js

```diff
@@ -30,6 +40,7 @@ const defaultFormat = process.env.SCREENIE_DEFAULT_FORMAT || 'jpeg';
  * signal. Exit with status code 143 (128 + SIGTERM's signal number, 15).
  */
 process.on('SIGTERM', () => {
+    logger.verbose('Received SIGTERM, exiting...');
```

Comment:
Suggest this to be `INFO` as well, similar to how we log startup message as `INFO`.